website/thaliawebsite/settings.py

```diff
@@ -683,20 +683,19 @@ def from_env(
 
 # TinyMCE config
 TINYMCE_DEFAULT_CONFIG = {
-    "selector": "textarea",
-    "plugins": "advlist anchor autolink autosave charmap code codesample fullscreen hr image media link lists table",
-    "toolbar1": "styleselect | bold italic underline strikethrough | hr charmap codesample | link unlink | bullist numlist outdent indent | table image media | undo redo | code",
-    "contextmenu": "bold italic underline strikethrough | image",
+    "height": 500,
     "menubar": False,
-    "inline": False,
-    "statusbar": True,
-    "width": "auto",
-    "height": 240,
```

Comment:
Auto width no longer required? Is 500 not really large on some admin pages?

Reply:
Auto width is not required, it will automatically fill. 
I figured 240px was quite small (which made sense because earlier, we had both dutch and English fields, but now we only have one).